index.html

```diff
@@ -7,40 +7,75 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
     .header {
-      text-align: center;
+      text-align: left;
```

Comment:
Understood. Note that there's no obligation to respond to comments before class. We'll continue to iterate on this project next week, and so the comments are intended to guide that iteration.